components/tabstrip/persist-content.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Persist Content
+page_title: Persist Content
+description: Persist Content of the TabStrip for Blazor.
+slug: tabstrip-persist-content
+tags: telerik,blazor,tab,strip,tabstrip,overview
+published: True
+position: 10
+---
+
+# Persist Content
+
+By default, the content of the TabStrip is rendered in the DOM when the Tab is active. Once it loses focus, its content is disposed and re-initialized again when the user selects the corresponding tab.
+
+If you want to change this behavior and keep the TabStrip content in the DOM while the user browses through the tabs, you can achieve that by enabling the `PersistContent` parameter the TabStrip exposes. It takes a `bool` and when set to `true`, the TabStrip content of the inactive tabs will be preserved and only hidden with CSS.
```

Comment:
add a paragraph to note that this will initialize all tabs on initialization of the tab strip, which is not the case before

Reply:
It will not initialize all tabs' content on initialization. The behavior will be kept as before - only the active tab (if any) will be initialized, others will be initialized when activated but not disposed when deactivated.